src/Microsoft.OpenApi.OData.Reader/Edm/EdmTypeExtensions.cs

```diff
@@ -0,0 +1,35 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// ------------------------------------------------------------
+
+using Microsoft.OData.Edm;
+
+namespace Microsoft.OpenApi.OData.Edm
+{
+	/// <summary>
+    /// Extension methods for <see cref="IEdmType"/>
+    /// </summary>
+    public static class EdmTypeExtensions
+	{
+		/// <summary>
+		/// Determines wether a path parameter should be wrapped in quotes based on the type of the parameter.
+		/// </summary>
+		/// <param name="edmType">The type of the parameter.</param>
```

Comment:
format

Reply:
Sorry, what's the issue with the format here?

Reply:
alignment, thanks.

Reply:
are you referring to the use of tabs vs spaces? still trying to understand what you think needs to be changed here.

Reply:
Yes, replace tab using 4 spaces.

Reply:
done!